docs/1-environment-setup.md

```diff
@@ -22,6 +22,19 @@ tips provided for a local workflow using [Visual Studio Code](https://code.visua
 
 ## Creating your project
 
+### Using Codespaces (Recommended)
+
+1. To get started, create a new project using the example project Sphinxy as a template
+   [on codespaces](https://codespaces.new/aelsayed95/sphinxy). This will open up a copy of Sphinxy
+   on codespaces for you.
```

Comment:
btw - it only takes `main` when you create a project from template into codespaces